client/src/pages/Login/functions.jsx

```diff
@@ -0,0 +1,19 @@
+export async function login(username, password) {
+  console.log(username, password);
+  /*eslint no-undef: 0*/
+  let promise = new Promise((res, rej) => {
```

Comment:
@natapokie can you send me a list of all the places (like this) in this PR where we should be calling to an API but currently are not?

Reply:
@Cali0707 the functions in this file, `login `and `resetPassword `need to be calling to the email api